src/main/java/me/thegiggitybyte/wumpuslink/WumpusLink.java

```diff
@@ -18,22 +17,14 @@
 import java.util.UUID;
 
 public class WumpusLink implements DedicatedServerModInitializer {
-    private static final String[] REQUIRED_CONFIG_KEYS;
-    private static SimpleConfig config;
 
     static { // Our initialization needs to be completed before the Fabric loader starts its own initialization process.
-        REQUIRED_CONFIG_KEYS = new String[]{
-                "discord-bot-token",
-                "discord-channel-id",
-                "discord-webhook-url"
-        };
-
         WumpusLink.initialize();
 
-        boolean canSendStatusMessages = WumpusLink.getConfig().getOrDefault("minecraft-server-status-messages", true);
+        JsonPrimitive canSendStatusMessages = JsonConfiguration.getUserInstance().getValue("minecraft-server-status-messages");
```

Comment:
This variable should be a `boolean` and the call to `JsonConfiguration.getValue()` should have `getAsBoolean()` appended to it; avoid explicitly using `JsonPrimitive` where possible!